config.yml

```diff
@@ -0,0 +1,112 @@
+defaults:
+  index:
```

Comment:
let's call this section `app` like we do in queue...

Then elements of loader thing is easier to copy/paste between components... it's more generic..